libs/designsystem/src/lib/scss/themes/_colors.scss

```diff
@@ -10,6 +10,15 @@ $notification-colors: (
   'danger': #ff595e,
 );
 
+$notification-and-info-colors: (
```

Comment:
I may be a little late to the party as I haven't heard the preceding discussions, but nevertheless :)
Generally, naming like `somethingANDsomethingelse` represents bit of a code smell to me.
In this case, is `info` not a type of notification? If yes, I think it should be added to the existing map

Reply:
Thank you for your input, @flemcito 👍 

Extending the existing `$notification-colors` map was the initial plan, but after inspecting the codebase we decided to go with this solution. Sometimes _notification_ is limited to _traffic light colors_ (`success`, `warning`, `danger`) and sometimes `info` is also an option. It may be a candidate for a refactoring issue to streamline the use of notification colors.

Reply:
Makes sense 👍 
Maybe`$notification-with-info-colors` better reflects that notifications are extended?